src/app/common/components/bg-badgecard.ts

```diff
@@ -139,18 +152,13 @@ export class BgBadgecard {
 	checked = false;
 
 	@HostBinding('class') get hostClasses(): string {
-		return this.checked
-			? 'tw-bg-[var(--color-lightgreen)]'
-			: 'tw-bg-white';
+		return this.checked ? 'tw-bg-[var(--color-lightgreen)]' : 'tw-bg-white';
 	}
 	@HostBinding('class') get completedClass(): string {
-		return this.completed
-			? 'tw-bg-[var(--color-green)]'
-			: 'tw-bg-white';
+		return this.completed ? 'tw-bg-[var(--color-green)]' : 'tw-bg-white';
 	}
 
-
-	showCompetencies = false
+	showCompetencies = false;
```

Comment:
I'm just gonna assume that nothing except for formatting and similar changed in this file